Marlin/ultralcd_impl_DOGM.h

```diff
@@ -522,13 +504,16 @@ static void lcd_implementation_status_screen() {
     uint8_t n = LCD_WIDTH - (START_COL) - 2;
 
     lcd_implementation_mark_as_selected(row, isSelected);
-
+#if 1
+      lcd_printstr_P (pstr, PIXEL_LEN_NOLIMIT);
+#else
     while (c = pgm_read_byte(pstr)) {
       n -= lcd_print(c);
       pstr++;
     }
     while (n--) lcd_print(' ');
-    u8g.setPrintPos(LCD_PIXEL_WIDTH - (DOG_CHAR_WIDTH), (row + 1) * (DOG_CHAR_HEIGHT));
+#endif
```

Comment:
If replacing lines of code, you should just replace them. Don't bother with these faux conditional blocks.